test/suites/migration.sh

```diff
@@ -424,6 +424,16 @@ migration() {
   # Test stateless copies
   lxc_remote copy --stateless l2:migratee/snap0 l1:migratee-new-name
 
+  if [ "${lxd_backend}" = "btrfs" ] && [ "${lxd2_backend}" = "btrfs" ] || [ "${lxd_backend}" = "zfs" ] && [ "${lxd2_backend}" = "zfs" ]; then
```

Comment:
Why dont we do this test for all storage types (I realise only the btrfs and zfs ones will do optimized transfers, but would also test cross-remote copies for all storage drivers right) ?